W-14659026-SFDC-Bridge-se (#359)
mulesoft/docs-rpa: rpa-manager/modules/ROOT/pages/processoperations-deploymentmap.adoc

You must manually keep configuration changes in RPA Manager and Anypoint Exchange in sync. Configurations might become out of sync in the following situations:

> * Invocable run configurations deleted in RPA Manager might still exist in Anypoint Exchange, but the bot cannot run them and they do not appear in RPA Manager.

Reviewer: ```suggestion
* Invocable run configurations deleted in RPA Manager might still exist in Anypoint Exchange, but the bot can't run them and they don't appear in RPA Manager.
```

Author: Replaced in all rpa-docs

* You can only view processes for which you are part of the process team.
* To execute, pause, or stop configurations, obtain the needed team member permissions for the respective phase.

> * You cannot interact with invocable configurations.

Reviewer: ```suggestion
* You can't interact with invocable configurations.
```
